src/main/java/io/fabric8/maven/docker/StartMojo.java

```diff
@@ -331,6 +336,26 @@ public StartedContainer call() throws Exception {
         return imagesWaitingToStart;
     }
 
+    private void createVolumesAsPerVolumeBinds(ServiceHub hub, List<String> volumeBinds, List<VolumeConfiguration> volumesConfigs)
```

Comment:
can we move this out of the StartMojo into the RunService ? The Mojo should stay as slim as possible.